prime-router/src/main/kotlin/fhirengine/translation/HL7toFhirTranslator.kt

```diff
@@ -16,36 +17,16 @@ import org.hl7.fhir.r4.model.Bundle
 /**
  * Translate an HL7 message to FHIR.
  */
-class HL7toFhirTranslator internal constructor(
+class HL7toFhirTranslator(
+    private val configFolderPath: String = "./metadata/fhir_mapping",
```

Comment:
Instead of a singleton instance that is used globally, the caller should now create a HL7toFhirTranslator instance to call the translate function. There should be no change in behavior if constructed without parameters.